frontend/src/components/auth/Registration.jsx

```diff
@@ -5,33 +5,45 @@ import { Link, useNavigate } from "react-router-dom";
 export default function Registration() {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  // Set useState for isTeacher with a default value of false
```

Comment:
when leaving code comments, comment things that are not obvious at first glance. This doesn't need to be commented.

```suggestion
```